src/libraries/System.IO.Compression/src/System/IO/Compression/ZipHelper.cs

```diff
@@ -189,5 +190,15 @@ private static bool SeekBackwardsAndRead(Stream stream, byte[] buffer, out int b
                 return true;
             }
         }
+
+        internal static bool GetSuppressValidation()
```

Comment:
(nit) I don't think this needs to be `internal`. It can be `private`.
(nit) Could maybe rename this method to `GetSuppressHeaderValidation()`.